### PR TITLE
Add handling zk://./path

### PR DESCRIPTION
**Pull request by bochkver**

### CODE REVIEW

**Comment by bochkver** [comment]
@ateska Should it be documented as an another case in docstring?

**Comment by ateska** [comment]
> @ateska Should it be documented as an another case in docstring?

Yes, of course.

File: asab/library/providers/zookeeper.py [hunk 1]
```diff
@@ -111,6 +111,10 @@ def __init__(self, library, path):
 		if z_url is None and url_pieces.netloc == "" and url_pieces.path == "" and self.Zookeeper.Path != '':
 			self.BasePath = '/' + self.Zookeeper.Path
 
+		# Handle `zk://./path` configuration
+		if z_url is None and url_pieces.netloc == "." and self.Zookeeper.Path != '':
```
Comment (ateska) [inline asab/library/providers/zookeeper.py:115]: `url_pieces.netloc.startswith(".")` ?

**Review by ateska — CHANGES_REQUESTED** [review]

File: asab/library/providers/zookeeper.py (hunk 1 above)
Comment (bochkver) [reply asab/library/providers/zookeeper.py:115]: It doesn't starts with `'.'`  

`url_pieces = ParseResult(scheme='zk', netloc='.', path='/path' ...`

File: asab/library/providers/zookeeper.py [hunk 2]
```diff
@@ -90,8 +108,8 @@ def __init__(self, library, path):
 		if self.BasePath == '/':
 			self.BasePath = ''
 
-		if url_pieces.netloc == "":
-			# if netloc is not providede `zk:///path`, then use `zookeeper` section from config
+		if url_pieces.netloc == "" or ".":
```
Comment (ateska) [inline asab/library/providers/zookeeper.py:111]: This seems to be weird ... do `url_pieces.netloc in {"", "."}`

**Review by ateska — CHANGES_REQUESTED** [review]

**Review by ateska — APPROVED** [review]

**Pull request merged**